test/api-digester/source-stability.swift.expected

```diff
@@ -7,6 +7,9 @@ Protocol IndexableBase has been removed (deprecated)
 Protocol MutableIndexable has been removed (deprecated)
 Protocol RandomAccessIndexable has been removed (deprecated)
 Protocol RangeReplaceableIndexable has been removed (deprecated)
+Func Array.append(contentsOf:) has been removed
+Func ArraySlice.append(contentsOf:) has been removed
+Func ContiguousArray.append(contentsOf:) has been removed
```

Comment:
No need for migration help, these are specialized versions for collection, when they're removed, user code will just revert to passing their collections into the sequence version.